libs/libc/math/lib_modff.c

```diff
@@ -44,12 +44,12 @@ float modff(float x, float *iptr)
     }
   else if (fabsf(x) < 1.0F)
     {
-      *iptr = (x * 0.0F);
+      *iptr = 0.0F;
       return x;
     }
   else
     {
-      *iptr = (float)(int)x;
+      *iptr = (float)(int) x;
```

Comment:
why add space before x?

Reply:
Just to keep the same style as in other variants of `modf`. I can remove spaces in other two variants instead.

Reply:
most cast in NuttX doesn't add space after cast, so I suggest to follow this convention

Reply:
Done